superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -4070,52 +4066,52 @@ msgstr "Editor de SQL"
 
 #: superset/views/core.py:2414 superset/views/core.py:2423
 msgid "SQL Lab"
-msgstr "Laboratório SQL"
+msgstr "SQL Lab"
 
 #: superset/views/core.py:2418
 msgid "Query Search"
-msgstr "Pesquisa de consulta"
+msgstr "Buscar Query"
```

Comment:
Personally still prefer "Pesquisa de consulta" ou "Pesquisa Query"

Reply:
I agree, is technically correct. It will depends the user, if the target are some sort of technical Query is the best, for end user the best would be Consulta. What do you think?